index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Microverse Portfolio project</title>
+  </head>
+  <body>
+    <header>
+      <div class="Logo">
+        <h4><a href="#"></a>Solomon Afutu</h4>
+        <nav>
+          <ul class="Menu-bar">
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <section class="Header-main">
+      <h1>
+        I’m Solomon<br />
+        Glad to see <br />
+        you!
+      </h1>
+      <p>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <h3>LET'S CONNECT</h3>
+      <ul>
+        <a href="https://twitter.com/SolomonAfutu"
+          ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
+        ></a>
+        <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+          ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
+        ></a>
+        <a href="https://github.com/SolomonTheCoder"
+          ><i class="fa fa-github fa-2x" aria-hidden="true"></i
+        ></a>
+        <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+        <i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i>
+      </ul>
```

Comment:
- [ ] You have done great well using the `li` and `ul` tags to implement the link icons 👏 👏 . However, there is also a need to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=tFTlxlBLvFLIXvcT-0). For this reason, I suggest that you ensure that the `ul` is along the same alignment as the rest of the features. 
- [ ] In addition, I kindly request to use the exact icon used in Figma template design
- [ ] Finally, humbly consider increasing the space between the icons. This will make the project look more professional 😎 

Your design 😎 
![links](https://user-images.githubusercontent.com/99394374/218763871-fe3391fc-5696-4131-9e79-942757446cff.png)

Figma design 😎 
![links fig](https://user-images.githubusercontent.com/99394374/218763965-3b3e91e5-cce6-43d5-98ab-aa0cc81cc91b.png)